.github/workflows/build.yml

```diff
@@ -56,6 +56,24 @@ jobs:
           destination-repository-name: 'fastasyncvoxelsniper-javadocs'
           user-email: ${{ secrets.USER_EMAIL }}
           target-branch: main
+      - name: Publish to CurseForge
+        if: ${{ runner.os == 'Linux' && env.STATUS == 'release' && github.event_name == 'push' && github.ref == 'refs/heads/main'}}
+        uses: Kir-Antipov/mc-publish@v3.2
+        with:
+          curseforge-id: 454430
+          curseforge-token: ${{ secrets.CURSEFORGE_TOKEN }}
+          files-primary: build/libs/!(*-@(sources|javadoc)).jar
+          name: FastAsyncVoxelSniper ${{ github.event.release.tag_name }}
+          version: ${{ github.event.release.tag_name }}
+          version-type: release
+          game-versions : |
+            1.19.1
```

Comment:
Just noticed, 1.19.2 should go here.

Reply:
![Screenshot 2022-10-03 at 09 33 40](https://user-images.githubusercontent.com/13383509/193523612-7adfb1e6-f8f8-4bd6-b897-4026694b23d5.png)
CF has a pretty selective list of versions